domain/domain.go

```diff
@@ -1407,6 +1403,10 @@ func (do *Domain) syncIndexUsageWorker(owner owner.Manager) {
 }
 
 func (do *Domain) updateStatsWorker(ctx sessionctx.Context, owner owner.Manager) {
+	// Negative stats lease indicates that it is in test, it does not need update.
+	if do.statsLease >= 0 {
+		do.loadStatsWorker()
```

Comment:
The function of `loadStatsWorker` also has a `for` infinite loop. After this PR, how to run the following logic?

Reply:
I have refactored this method of fixing. PTAL.